.github/workflows/reusable-wait-block-target.yml

```diff
@@ -19,6 +19,9 @@ on:
         required: false
         type: boolean
         default: false
+      TARGET_NAME:
```

Comment:
we might need to give the job a unique name according to input.TARGET_NAME so it's easy to distinguish in the PR checks.
[Here](https://github.com/earthly/earthly/blob/f8b0db17a9a789ee2b22edbdd38684ac0622e7b1/.github/workflows/reusable-example.yml#L42) is an example where this is done.

Reply:
Good idea. How about something like this? https://github.com/earthly/earthly/pull/3355/commits/4d04a40e5a7b65f7b52b84ceb0a0a4042580fefa